hi-ml-cpath/src/health_cpath/utils/plots_utils.py

```diff
@@ -37,21 +39,43 @@ def validate_class_names_for_plot_options(
     return class_names
 
 
-def save_scores_histogram(results: ResultsType, figures_dir: Path) -> None:
+def save_scores_histogram(results: ResultsType, figures_dir: Path, stage: str = '') -> None:
     """Plots and saves histogram scores figure in its dedicated directory.
 
     :param results: List that contains slide_level dicts
     :param figures_dir: The path to the directory where to save the histogram scores.
+    :param stage: Test or validation, used to name the figure
```

Comment:
could we add default value for stage in the docstring?

Reply:
done